### PR TITLE
feat(vscode): support input variables in launch config

### PR DESCRIPTION
**Pull request by ykageyama-mondo**
This PR adds support for adding input variables to the vscode launch config.

This will allow the creation of more complex debugger configurations.

See https://code.visualstudio.com/docs/editor/variables-reference#_input-variables

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/vscode/launch-config.ts [hunk 1]
```diff
@@ -109,6 +148,7 @@ export class VsCodeLaunchConfig extends Component {
     this.content = {
       version: "0.2.0",
       configurations: [],
+      inputs: [],
```
Comment (mrgrain) [inline src/vscode/launch-config.ts:151]: I know you are mirroring how `configurations` work, but can we keep this out of an "empty" config please?

File: src/vscode/launch-config.ts [hunk 2]
```diff
@@ -130,9 +170,59 @@ export class VsCodeLaunchConfig extends Component {
...
   }
+
+  /**
+   * Adds an input variable with type `promptString` to `.vscode/launch.json`.
+   *
+   * See https://code.visualstudio.com/docs/editor/variables-reference#_input-variables for details.
+   * @param cfg VsCodeLaunchPromptStringInputEntry
+   */
+  public addPromptStringInput(cfg: VsCodeLaunchPromptStringInputEntry) {
+    this.content.inputs.push({
```
Comment (mrgrain) [inline src/vscode/launch-config.ts:181]: Something like this should work:
```suggestion
    this.content.inputs ??= [];
    this.content.inputs.push({
```

**Comment by ykageyama-mondo** [comment]
Thanks for the feedback! I've made some changes to make the `inputs` property optional.

File: .vscode/launch.json [hunk 3]
```diff
@@ -14,5 +14,6 @@
       ]
     }
   ],
+  "inputs": [],
```
Comment (mrgrain) [inline .vscode/launch.json:17]:
```suggestion
```

**Review by mrgrain — APPROVED** [review]

**Pull request merged**